src/plugins/sessions/index.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  JupyterFrontEnd,
+  JupyterFrontEndPlugin
+} from '@jupyterlab/application';
+
+/**
+ * A plugin to open documents in the main area.
+ */
+const plugin: JupyterFrontEndPlugin<void> = {
+  id: 'quetz:stop-polling',
+  autoStart: true,
+  activate: (app: JupyterFrontEnd): void => {
+    app.serviceManager.sessions?.ready.then(value => {
+      // stop polling the kernel sessions
+      app.serviceManager.sessions['_kernelManager']['_pollModels']?.stop();
+      // stop polling the sessions
+      void app.serviceManager.sessions['_pollModels'].stop();
+    });
+
+    app.serviceManager.kernelspecs?.ready.then(value => {
+      // stop polling the kernelspecs
+      void app.serviceManager.kernelspecs.dispose();
+    });
+
+    /* 
+    app.serviceManager.terminals?.ready.then( value => {
+      console.debug("Stopping terminals:");
+      // stop polling the terminals
+      void app.serviceManager.terminals['_pollModels'].stop();
+    });
```

Comment:
To uncomment?

Reply:
No, it's commented because the terminal session is not initialized so if we try to access them there is an error.

Reply:
I leave it commented just in case we need to use it later.

Reply:
Ah, it might because the `terminal_available` option is not passed in the page config, so the polling never starts.

We can leave it like this then :+1: